### PR TITLE
Made slightly more useful.

### PR DESCRIPTION
**Pull request by jbone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 
-# Nothing to see here, move along now.
+# Decentralized activity streams and more.
```
Comment (pjz) [inline README.md:2]: I think you should add 'using open standards' here.

Comment (jbone) [inline README.md:2]: Goes without saying. ;-)

Comment (jbone) [inline README.md:2]: And possibly not true.

Comment (pjz) [inline README.md:2]: Well, needs more text of _some kind_... oh! we definitely need a LICENSE file as well, or maybe just here say 'under whatever licesne'

**Pull request merged**